### PR TITLE
Feature/fix review stuff

### PR DESCRIPTION
**Pull request by jespersandnielsen**

### CODE REVIEW

**Comment by jespersandnielsen** [comment]
ping @jacobtonder

File: main/src/ca/uwaterloo/flix/language/phase/jvm/GenSpawnClasses.scala [hunk 1]
```diff
@@ -14,35 +14,40 @@ object GenSpawnClasses {
...
     // Class visitor
     val visitor = AsmOps.mkClassWriter()
 
-    // Class header.
+    // Class visitor.
     visitor.visit(AsmOps.JavaVersion, ACC_PUBLIC + ACC_FINAL, JvmName.Spawn.toInternalName, null,
       JvmName.Object.toInternalName, Array(JvmName.Runnable.toInternalName))
 
-    visitor.visitSource(JvmName.Spawn.name, null)
+    // visitor.visitSource(JvmName.Spawn.name, null)
```
Comment (jacobtonder) [inline main/src/ca/uwaterloo/flix/language/phase/jvm/GenSpawnClasses.scala]: Remove if not needed

File: main/src/ca/uwaterloo/flix/runtime/interpreter/Interpreter.scala [hunk 2]
```diff
@@ -419,42 +439,67 @@ object Interpreter {
...
+   */
+  private def unlockAllChannels(channels: List[Value.Channel]): scala.Unit = {
     for (chan <- channels) {
       chan.lock.unlock()
     }
+  }
 
-  private def addConditions(channels: List[Value.Channel], lock: Lock, condition: Condition): scala.Unit =
+  /**
+   * Add consition to all channels.
```
Comment (jacobtonder) [inline main/src/ca/uwaterloo/flix/runtime/interpreter/Interpreter.scala]: ~~consition~~ condition

**Review by jacobtonder — CHANGES_REQUESTED** [review]
A few minor things.

I haven't proof read the tests, but it looks fine.

**Comment by jespersandnielsen** [comment]
@jacobtonder please review again

**Review by jacobtonder — APPROVED** [review]

**Pull request merged**